plotly/widgets/graph_widget.py

```diff
@@ -247,6 +246,18 @@ def message_handler(widget, ranges):
         """
         self._handle_registration('zoom', callback, remove)
 
+    def plot(self, figure_or_data, validate=True):
+        """Plot figure_or_data in the Plotly graph.
+        """
+        figure = tools.return_figure_from_figure_or_data(figure_or_data, validate)
+        message = {
+            'task': 'newPlot',
+            'data': figure['data'],
+            'layout': figure.get('layout', {}),
+            'graphId': self._graphId
+        }
+        self._handle_outgoing_message(message)
+
```

Comment:
Should this support refreshing a blank graph? I.e., should this work?

`g.plot()`

or *at least*

`g.plot({})`

Does validation mess with that or something? It seems like it would mirror the `g = GraphWidget()` initialization well.